posts/coalesce2022-a-recap.md

```diff
@@ -0,0 +1,238 @@
+
+19. Make decisions self-service fo ryour business
+> Problem: How do you ensure the business is making data-driven decisions in a self-service manner?
+> - Can I trust this data?
+> - What does the metric mean?
+> - Who can I reach out to about the data/metric?
+> - Can I run a quick filter/pivot on this metric?
+> 
+> Solution: You can integrate Atlan into your BI tool to show all of the metric context from the dbt Semantic layer right in the dashboard. So you have all of the data context to make decisions
+
+20. Enable data teams that can change the world
+> Problem: How do you unify and enable a diverse team to facilitate data to create change?
+> 
+> Solution: Embrace the diversity in a team. It is a data team's biggest strenght and weakness. Atlan's metadata is what can help a team gel with each other and do great things.
+
+[Watch Prukalpa's Full Talk here](https://www.youtube.com/watch?v=fxFxfKCZrhY&list=PL0QYlrC86xQlj9UDGiEwhXQuSjuSyPJHl&index=78)
+[For more information on all that Atlan can offer](https://atlan.com/)
+
+## Detecting Data Anomalies via an Inspection Layer
+In most of our current data ecosystems, we ingest data from various vendors, IoT devices, and more. Unfortunately, for every data source that is perfect, there is another (and we've all been there) where the data isn't what we were expecting. When the data has nulls, contains duplicates, or has a different schema than initially communicated, your data pipeline can stumble. You can handle this natively with dbt native methods and implement an inspection layer to ensure that 'bad' data is flagged and quarantined while the remainder is loaded uninterrupted.
+
+Neal Achord and John Rensberger from Red Pill Analytics walk us through their implementation of the inspection layer in this workshop. It was fascinating to see and implement this myself.
+
+You can follow along with the workshop by [watching their talk](https://www.youtube.com/watch?v=EKPi-T_nBio&t=80s), cloning the [repo](https://github.com/RedPillAnalytics/dbt_coalesce_workshop) on Github and referring to their [slides](https://docs.google.com/presentation/d/11Q9wwMfyz6xuxMXCPizFg4DKSY_zOIHPNOrsNI8oBn8/edit?usp=sharing).
+
+___
+I'm cognizant that all of the above is a lot. Thanks so much for sticking with me as I condensed a jam-packed week of content into a short little blog like this. I had a ton of fun and while I felt like I was drinking from a firehose at times, Coalesce was the best experience I've had at a conference to date. I thoroughly enjoyed getting to chat with data professionals and commiserating over all of the common data problems we face daily. All of their sessions and workshops where immensely thought-provoking and interesting. I hope I'll get to attend again sometime soon!
```

Comment:
Definitely a lot!!! Great write up. Thank you for bringing this recap to what seemed like a very impactful conference.  I absolutely love your writing style and think that your personality really showed through (a.k.a. not just an emotionless recap spewed on the page).

Reply:
Thanks!